apps/condo/domains/meter/components/Import/Index.tsx

```diff
@@ -0,0 +1,173 @@
+import isFunction from 'lodash/isFunction'
+import React, { useCallback, useEffect, useMemo, useRef, useState } from 'react'
+import XLSX from 'xlsx'
+
+import { useIntl } from '@open-condo/next/intl'
+import { Button } from '@open-condo/ui'
+
+import { BaseImportWrapper } from '@condo/domains/common/components/Import/BaseImportWrapper'
+import {
+    fitToColumn,
+    IImportWrapperProps,
+    IMPORT_EVENT,
+    ImportEmitter,
+} from '@condo/domains/common/components/Import/Index'
+import { TrackingEventType, useTracking } from '@condo/domains/common/components/TrackingContext'
+import { MetersDataImporter } from '@condo/domains/meter/components/MetersDataImporter'
+import { TOnMetersUpload } from '@condo/domains/meter/components/MetersDataImporterTypes'
+import { useMetersImporter } from '@condo/domains/meter/hooks/useMetersImporter'
+import { ProcessedRow } from '@condo/domains/meter/utils/metersImporters/AbstractMetersImporter'
+
+export type IMetersImportWrapperProps = Pick<IImportWrapperProps, 'accessCheck' | 'onFinish' | 'uploadButtonLabel' | 'importCardButton'>
+
+type ActiveModalType = null | 'example' | 'progress' | 'partlyLoaded' | 'success' | 'error'
+
+const MetersImportWrapper: React.FC<IMetersImportWrapperProps> = (props) => {
```

Comment:
why can not reuse ImportWrapper from common domain?

Reply:
Refactored